src/main/java/edu/hm/hafner/analysis/IssueBuilder.java

```diff
@@ -553,6 +560,22 @@ public Optional<Issue> buildOptional() {
         return Optional.of(buildAndClean());
     }
 
+    private boolean isAbsolutePath(final String stringPath) {
```

Comment:
Would the implementation of `PathUtil#isAbsolute` work? Or `FilenameUtils.getPrefixLength(fileName) > 0`? `PathUtil` is already used in the class and can be extracted as a CONSTANT.

Reply:
Ok.